esmvaltool/references/noaa-mbl-ch4.bibtex

```diff
@@ -0,0 +1,9 @@
+@misc{noaa-mbl-ch4,
+        doi = {https://doi.org/10.15138/P8XG-AA10},
+	url = {https://gml.noaa.gov/ccgg/trends_ch4/},
+	year = 2023,
+        month = {jul},
+	author = {Lan, X., K.W. Thoning, and E.J. Dlugokencky},
+	title = {Trends in globally-averaged CH4, N2O, and SF6 determined from NOAA Global Monitoring Laboratory measurements.},
```

Comment:
On the website they have added also a version to the citation. At the moment it is at "Version 2023-08". Not sure if we have a specific identidyer for that, or if we should just add it to the title?

Reply:
I was uncertain adding the version because I believe it depends on the actually downloaded file. Should we add the current one anyway? Or is there a possibility to detect the version?

Reply:
I would add the current version here, since the citation is very specific on the NOAA website on how to cite the file that you can download. Not sure if there would be a possibility to detect the version automatically. If it is not in the name of the csv-file that is downloaded it might be tricky.